npm-shrinkwrap.json

```diff
@@ -5,8 +5,8 @@
   "node-version": "v0.10.38",
   "dependencies": {
     "babel-core": {
-      "version": "5.6.12",
-      "resolved": "https://registry.npmjs.org/babel-core/-/babel-core-5.6.12.tgz",
+      "version": "5.6.15",
+      "resolved": "https://registry.npmjs.org/babel-core/-/babel-core-5.6.15.tgz",
```

Comment:
Can we avoid updating other dependencies unless the task for the PR explicitly requires this?
I love upgrading, but I want to make sure that we look through change logs before doing any upgrades, to makes sure there are no breaking changes.